src/framework/auth/services/token.service.ts

```diff
@@ -71,6 +80,18 @@ export class NbAuthJWTToken extends NbAuthSimpleToken {
 
     return date;
   }
+
+  /**
+   * Is data expired
+   * @returns {boolean}
+   */
+  isValid(): boolean {
+    const expDate: Date = this.getTokenExpDate();
```

Comment:
could be much simpler as `return super.isValid() && new Date() < this.getTokenExpDate();`

Reply:
fixed